packages/database/src/migrations/20210607052816-CreatePostCodeEntityType-modifies-schema.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import { TupaiaDatabase } from '../TupaiaDatabase';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function () {
+  const db = new TupaiaDatabase();
+  await db.executeSql(`
+    ALTER TYPE public.entity_type ADD VALUE 'postcode';
+  `);
+  return db.closeConnections();
+};
```

Comment:
```suggestion
exports.up = async function (db) {
  await db.executeSql(`
    ALTER TYPE public.entity_type ADD VALUE 'postcode';
  `);
};
```
Is there a reason we aren't doing this migration the normal way?

Reply:
tbh I don't know, but I think that is our traditional way to add an `entity.type` 😁